src/openrct2/actions/ParkSetParameterAction.hpp

```diff
@@ -40,6 +40,12 @@ DEFINE_GAME_ACTION(ParkSetParameterAction, GAME_COMMAND_SET_PARK_OPEN, GameActio
     {
     }
 
+    void AcceptParameters(GameActionParameterVisitor & visitor) override
+    {
+        visitor.Visit("parameter", _parameter);
```

Comment:
this action does not follow the order of other commands. value should be parameter and parameter should be some sort of type